#171648160 CHORE: Setting up eslint to the project

brianineza01 opened the pull request:
#### what does this PR do?
this PR sets up eslint to the project
#### Pivotal tracker story Id?
#171648160

deschantkn reviewed (changes requested):
Good job with setting up ESLint @brianineza01 . However I have a challenge for you to this further.
We need to have a consistent Pull Request template in the team so for that I would like you to do the following:
1. Create a `.github` folder at the root directory
2. Inside the `.github` folder, create a `PULL_REQUEST_TEMPLATE.md` file
3. Inside the `PULL_REQUEST_TEMPLATE.md` file add the following content:

```
#### Description

#### Type of change

Please select the relevant option

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Others (cosmetics, styling, improvements)
- [ ] This change requires a documentation update

#### How Has This Been Tested?

- [ ] Unit
- [ ] Integration
- [ ] End-to-end
- [ ] No testing required

#### How to Test

Give instructions on how reviewers can verify your work

#### Any background context you want to add (Optional)

Screenshots or GIFs

#### STORY ID

[Finishes|Delivers|Fixes #ID]
```

N.B: Please add the `TTL Review` label on thie Pull Request. Finally you can reach out to me on Slack in case you have any questions regarding this task.

Thanks.

deschantkn reviewed (changes requested):
Awesome, you've added the PR Template. Now please add the TTL Review label and update your commit message as we discussed on slack.

brianineza01 committed:
171648160-story(eslint): add eslint config

- configure eslint with airbnb
- add .github folder with pull request template

[Finishes #171648160]

deschantkn reviewed (approved):
Good job @brianineza01. LGTM.
